src/components/search/SearchBar.tsx

```diff
@@ -19,6 +19,9 @@ export function SearchBar({ value, placeholder, onChangeValue, isFetching }: Pro
     onChangeValue(value);
   };
 
+  const iconStyle =
```

Comment:
Please move this to the bottom of the file into an object called `styles`. Here's an example: https://github.com/hyperlane-xyz/hyperlane-explorer/blob/main/src/components/nav/Footer.tsx#L93

Reply:
Done!)